README.md

```diff
@@ -1,46 +1,154 @@
-# Getting Started with Create React App
+<!-- PROJECT Header -->
+<br />
+<div align="center">
+
+  <h2 align="center">CRYPTO-TRACKING-APP
+</h3>
+
+  <p align="center">
+    Get All The Latest Info Regarding Your Favorite Crypto Currency And Crypto News Update!
+    <br />
+    <!-- <a href="https://github.com/othneildrew/Best-README-Template">View Demo</a> -->
+</div>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+        <li><a href="#wireframe">Wireframe</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+     <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
-## Available Scripts
 
-In the project directory, you can run:
 
-### `npm start`
+<!-- ABOUT THE PROJECT -->
+## About The Project
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+This app is build with the existing knowledge of the React basics, and give you further practice working with React context,Chart Js, Firebase,MUI, codebase linting, component testing,Git-Flow workflow , and TypeScript.
+ A user should, at the bare minimum, be able to view all the Top 100 list of Cryptocurrencys based on their market capital , Shows the trending coins over the last 24 hour,Search for a particular coin from the list and Shows a Historical chart of a specfic coin with a detail information on a daily ,monthly and year base and additionaly a user can learn and read on the latest update on the News section.
```

Comment:
I'd recommend making the list of features/functionalities a bulleted list for easier readability.